Lib/test/test_genericalias.py

```diff
@@ -293,5 +293,15 @@ def test_dir(self):
         for generic_alias_property in ("__origin__", "__args__", "__parameters__"):
             self.assertIn(generic_alias_property, dir_of_gen_alias)
 
+    def test_weakref(self):
+        for t in self.types:
+            if t is None:
+                continue
+            tname = t.__name__
```

Comment:
Having this in a variable seems unnecessary since it's only used once anyway

Reply:
Yup, I thought so too. I just did it to keep consistency with the other tests (test_subscriptable, test_unsubscriptable and test_instantiate do that).